low-level/files/opt/seagate/sspl/setup/resource_map/server.py

```diff
@@ -49,6 +48,8 @@ def __init__(self):
         self.server_frus = {
```

Comment:
this map can just have 'server' so later it can have mapping for sw services & OS resources too

Reply:
done